pkg/controllers/netpol/network_policy_controller.go

```diff
@@ -817,6 +817,14 @@ func (npc *NetworkPolicyController) syncPodFirewallChains(version string) (map[s
 			}
 		}
 
+		// add rule to log the packets that will be droopped due to network policy enforcement
+		comment = "rule to log dropped traffic POD name:" + pod.name + " namespace: " + pod.namespace
+		args = []string{"-m", "comment", "--comment", comment, "-j", "NFLOG", "--nflog-group", "100"}
```

Comment:
```suggestion
		args = []string{"-m", "comment", "--comment", comment, "-j", "NFLOG", "--nflog-group", "100", "-m", "limit", "--limit", "10/minute", "--limit-burst", "10"}
```

I recommend that we add some limiting here. I know that a few of our users put their kube-router instances on or near the edge and it would be good to ensure we don't get overwhelmed with failed packets.